src/locale/locales/fr/messages.po

```diff
@@ -4826,7 +4702,7 @@ msgstr "Post supprimé"
 
 #: src/lib/api/index.ts:161
 msgid "Post failed to upload. Please check your Internet connection and try again."
-msgstr ""
+msgstr "Échec de l’envoi du post. Vérifiez votre connexion Internet et réessayez."
```

Comment:
I thought about it, and I think that it would be a _little bit_ too polite to add a `Veuillez` here. I checked on other common translations in other software and it doesn't seem to be a good practice to add a `Veuillez` here, so I'll drop it in similar messages (I'll keep it when it's something a little bit more demanding to ask for).

Reply:
SGTM